cli/silence_add.go

```diff
@@ -56,8 +56,16 @@ var addCmd = &cobra.Command{
 }
 
 func init() {
-	user, _ := user.Current()
-	addCmd.Flags().StringP("author", "a", user.Username, "Username for CreatedBy field")
+	user, err := user.Current()
+	username := ""
+
+	if err != nil {
+		fmt.Errorf("Failed to get current user")
```

Comment:
This doesn't actually do anything -- it creates an `error` but doesn't indicate it to the user. It would be useful to let the user know that they don't have a default username available, and that they'll have to provide one (since we require an `author` when creating silences).

Reply:
right, fixed.